toolbox/array/concat-arrays.md

```diff
@@ -0,0 +1,33 @@
+# Overview
+
+![The Concat Arrays Node.](../../.gitbook/assets/toolbox/array/concat-arrays.png)
+
+**Concat Arrays** merges two or more **Arrays** into one new **Array**. The **Arrays** to be merged are given as input to the node. 
+
+Note that the **Arrays** that are to be merged by this node are not sorted.
+
+# Attributes
+
+|Attribute|Type|Description|
+|---|---|---|
+|`Default Count`|**Int**|The number of **Arrays** you wish to merge. The default number is 2 **Arrays**.|
+
+# Inputs
+
+|Input|Type|Description|
+|---|---|---|
+|*Pulse Input* (►)|**Pulse**|A standard **Input Pulse**, to trigger the execution of the **Node**.|
+|`Array` * (`Count` **Attribute**)|**Array**|Two or more **Input** **Arrays** you wish to merge.|
+
+# Outputs
+
+|Output|Type|Description|
+|---|---|---|
+|*Pulse Output* (►)|**Pulse**|A standard **Output Pulse**, to move onto the next **Node** along the **Logic Branch**, once this **Node** has finished its execution.|
+|`New Array`|**Array**|The merged **Array**.|
+
+# See Also
+- [**Array Value**](array-value.md)
+
+# External Links
+- [*Concatenation in computer programming*](https://en.wikipedia.org/wiki/Concatenation) on wikipedia.
```

Comment:
This links to an article about string concatenation.

Reply:
Array as well, mostly concatenation in general but I will move it specifically